extensions/azurecore/src/utils/TenantIgnoredError.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export class TenantIgnoredError implements Error {
+	name: string = 'TENANT_IGNORED_ERROR';
```

Comment:
Do you need this anymore? I don't see it being used anywhere.

Reply:
Since I implemented Error here, `name` came with it.